src/components/Title/Title.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface TitleProps {
+  title: React.ReactNode
+}
+
+export const Title = ({ title }: TitleProps): React.ReactElement => {
+  const classes = classnames('usa-logo')
+
+  return (
+    <div className={classes}>
+      <h1 className="usa-logo__text">{title}</h1>
```

Comment:
```suggestion
      link?
      <h1 className="usa-logo__text">
        <a href={link}>{title}</a>
      </h1>
      : <h1 className="usa-logo__text">{title}</h1>
```
Oops! My comment was unclear.  What you had before was good, just conditionally display the link. Only include it when it exists.  If it doesn't exist,  leave it out .

Reply:
I think we should avoid rendering `<a>` in general since we don't know what kind of routing a user is using. Also if we _do_ want to render `<a>` we need to allow for element attributes to be passed in